README.md

```diff
@@ -266,18 +270,18 @@ add the following options to the root command:
   --verbosity-level debug|info|verbose|quiet|silent  Specify the verbosity level. (default: info)
 ```
 
-A corresponding UI object is then available in the `Application` object. Whenever the
+A corresponding UI object is then available in the `Cli` object. Whenever the
 program wants to output something, it should use the `ui` object.
 
 ``` toit
-import cli
+import cli show Cli
 
-some-chatty-method app/cli.Application:
-  ui := app.ui
+some-chatty-method cli/Cli:
+  ui := cli.ui
   ui.debug "This is a debug message."
   ui.verbose "This is a verbose message."
```

Comment:
Maybe not the best verby name.

`verbose`, `interactive`, `result` are all kind of weird now - and got a bit weirder by the info -> inform, warning -> warn cleanup.

Reply:
I will do a follow-up PR.
As discussed in person:
```
emit --info
emit --warning
...
```